cucumber/api/features/retrieve_api_key.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Retrieving an API key with conjurctl
+
+  # We need to be in production environment to test this
+  Scenario: Retrieve an API key
+    Given I set environment variable "RAILS_ENV" to "production"
+    And I set environment variable "CONJUR_LOG_LEVEL" to "info"
```

Comment:
Why setting the conjur_log_level is relevant here?

Reply:
because if we are in debug mode the output will contain some junk. we change the env so we are in the expected environment of the client using this tool.

Reply:
o.k. Since its not suppose to influence the results.
In general i want to work with relevant steps only(in this case its not critical) because if from some reason we will fail on this step we cant say "retrieve an API key" is not working.